docs/static-sites.md

```diff
@@ -0,0 +1,223 @@
+# Static Websites
+### With S3, CloudFront and ACM
+
+A modern application pattern is to deploy static web content, say a React or Vue.js single page application, that interacts with an app API. For this example we want to deploy a static web site -- `https://www.gofaas.net` -- that talks to our gofaas API running at `https://api.gofaas.net` to authenticate a user and display a dashboard.
+
+This pattern offers many advantages. Now the API is only concerned with data, making it easier to write and more cost effective to run. The web content is completely static, making it extremely reliable and cost effective to deliver to our users.
+
+Compare this to a traditional Model View Controller (MVC) approach like Rails or Django. In this architecture the API may spend lots of time rendering HTML, and the HTML may not get served to users if there is an application bug or a database outage.
+
+Static websites are a solved problem on AWS. We simply create an S3 bucket configured for website hosting and upload the content with public-read permissions. Then anyone can access the content from a URL like `http://gofaas-webbucket-572007530218.s3-website-us-east-1.amazonaws.com` with some of the highest reliability and lowest storage and bandwidth costs possible.
+
+Serving this from a custom domain is also a solved problem. We add the CloudFront CDN, configured with an SSL cert via the AWS Certificate Manager, in front of the S3 bucket. When we point our custom domain DNS to CloudFront, users can access the content from a URL like `https://www.gofaas.net` with some of the fastest delivery times and lowest bandwidth costs possible thanks to the global content caching network.
```

Comment:
Need any information about invalidating cached objects during deploys?

Reply:
Good call. Also missing in the `make deploy-static` command